tests/util/test_unit_conversion.py

```diff
@@ -32,59 +35,31 @@
 )
 
 INVALID_SYMBOL = "bob"
+_CONVERTERS: list[type[BaseUnitConverter]] = [
+    obj
+    for _, obj in inspect.getmembers(unit_conversion)
+    if inspect.isclass(obj)
+    and issubclass(obj, BaseUnitConverter)
+    and obj != BaseUnitConverter
+]
+
+
+def _get_valid_unit(converter: type[BaseUnitConverter]) -> str:
+    """Get a valid unit from the converter, different from the normalized unit."""
+    return next(
+        filter(
+            lambda v: v != converter.NORMALIZED_UNIT,
+            sorted(converter.VALID_UNITS),
```

Comment:
Filtering and then sorting, means you need to sort fewer items, than sorting and then filtering.